poker-pals/src/routes.js

```diff
@@ -4,6 +4,7 @@ import profileComponent from './components/Profile.vue';
 import reportsComponent from './components/Reports.vue';
 import tablesComponent from './components/Tables.vue';
 import exampleComponent from './components/HelloWorld.vue';
+import bannedComponent from './components/Banned.vue';
 
 export default [
     {path: '/', redirect: { name: 'Login'}},
```

Comment:
will this cause the url to show up as `localhost:8080/#/login`? And if so, how does `replace` from other pages affect this url? I think it may be better to just load `loginComponent` at start instead of redirecting to it